frontend/assets/css/fonts.scss

```diff
@@ -277,6 +277,27 @@ h1 {
   font-weight: var(--button_font_weight);
 }
 
+
 .button_text{
   @include button_text
 }
+
+@mixin text-toast(){
+  font-family:var(--roboto-family);
+  font-size:var(--standard_text_font_size);
+  font-weight: var(--roboto-bold);
+}
+
+.text-toast{
+  @include text-toast
+}
```

Comment:
nice that you thought of putting them into `reusable classes and mixins`, but in my opinion, this would be an incentive to use the `styles` of a `toast` in any other component than a `toast`. Which would contradict best practices regarding `encapsulation` and `decoupling`.

Our way of abstraction should be: We need a `toast`, lets use our `BaseComponent` for `toasts`, which should define the `styles` to guarantee `cohesiveness`.

Reply:
Sounds great :rocket: